src/main/java/in/muthukumari/dao/ExistDAO.java

```diff
@@ -0,0 +1,190 @@
+package in.muthukumari.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import in.muthukumari.exception.DBException;
+import in.muthukumari.util.ConnectionUtil;
+
+public class ExistDAO {
+
+	private ExistDAO() {
+		// Default Constructor
+	}
+
+	/**
+	 * This method used to check if the usename is exists or not
+	 * 
+	 * @param userName
+	 * @return
+	 * @throws DBException
+	 */
+	public static boolean isExistUserName(String userName) throws DBException {
+		Connection con = null;
```

Comment:
declare con,pst,rs as class variable

Reply:
The variables scope is end at the method end so I declared it as local variable